Common/include/code_config.hpp

```diff
@@ -92,7 +92,7 @@ using su2double = codi::RealReversePrimal;
 #elif CODI_PRIMAL_INDEX_TAPE
 using su2double = codi::RealReversePrimalIndex;
 #else
-using su2double = codi::RealReverse;
+using su2double = codi::RealReverseIndex;
```

Comment:
by the way concerning these preprocessor statements? This is the code snippet from `code_config.hpp`
```c++
#if defined(CODI_REVERSE_TYPE) // reverse mode AD
#include "codi.hpp"
#include "codi/tools/dataStore.hpp"

#ifndef CODI_INDEX_TAPE
#define CODI_INDEX_TAPE 0
#endif
#ifndef CODI_PRIMAL_TAPE
#define CODI_PRIMAL_TAPE 0
#endif
#ifndef CODI_PRIMAL_INDEX_TAPE
#define CODI_PRIMAL_INDEX_TAPE 0
#endif

#if defined(HAVE_OMP)
using su2double = codi::RealReverseIndexParallel;
#else
#if CODI_INDEX_TAPE
using su2double = codi::RealReverseIndex;
#elif CODI_PRIMAL_TAPE
using su2double = codi::RealReversePrimal;
#elif CODI_PRIMAL_INDEX_TAPE
using su2double = codi::RealReversePrimalIndex;
#else
using su2double = codi::RealReverse;
#endif
#endif
```
In AD mode I can see in the build files that the flag `-DCODI_REVERSE_TYPE` is given so that code will be excecuted. Now all those three other vars `CODI_INDEX_TAPE ` `CODI_PRIMAL_TAPE` and `CODI_PRIMAL_INDEX_TAPE` are set to 0 (and none of those can be found in the build file ... or are #defined elsewhere) so in the following part will use the `#else` clause ... which you changed. Is it then the `JacobiTape` or now `JacobiIndexTape`?

Reply:
`codi::RealReverse` uses the `JacobiTape` and `codi::RealReverseIndex` uses the `JacobiIndexTape`. The change to `codi::RealReverseIndex` is a temporary way of using that type in the CI pipeline, this will be undone before merging.

Reply:
Ok so the `codi::RealReverseIndex` using the `JacobiIndexTape` is just for test purposes now? Sure, there is curiosity, but this change will be reverted: What is the idea with this test? (Or am I fully missing the point here)

I also guess that this RealReverse change introduced the failing regression test not the 3 updated Codi lines?

Reply:
... who are you and what have you done with the Tobi that was the meeting yesterday?
(Johannes is working on the issues the preaccumulation has with the RealReverseIndex type, which is one of the steps required to make it work with hybrid parallel AD) the codi lines are the fix, the failing tests are some other problem in the FEA solver)

Reply:
... attention span issues 🐔 😞 
but thanks for the explanation.